package/crds/postgresql.sql.crossplane.io_providerconfigs.yaml

```diff
@@ -70,6 +70,14 @@ spec:
                 default: postgres
                 description: Defines the database name used to set up a connection to the provided PostgreSQL instance. Same as PGDATABASE environment variable.
                 type: string
+              sslMode:
+                description: Defines the SSL mode used to set up a connection to the provided PostgreSQL instance
+                enum:
+                - disable
+                - require
+                - verify-ca
+                - verify-full
+                type: string
```

Comment:
The SSLMode is optional on the CRD, the client will take the `lib/pq` default, `require`: https://github.com/lib/pq/blob/caa87158f59995a15e2057d80d586391f51333f2/ssl.go#L19

Do you think we should default to that here too or keep relying on upstream (in case it changes we don't have to worry about updating it here)?

Reply:
Fair point; if we are falling back to the underlying default then I don't think we need to specify a default. Thanks for clarifying.